python/ngen_cal/tests/test_plugin_system.py

```diff
@@ -61,3 +69,11 @@ def ngen_cal_start(self) -> None:
     @hookimpl
     def ngen_cal_finish(self) -> None:
         """Called after exiting the calibration loop."""
+
+    @hookimpl
+    def ngen_cal_model_output(self) -> None:
+        """Test model output plugin"""
+
+    @hookimpl
+    def ngen_cal_model_post_iteration(self, path: Path, iteration: int) -> None:
```

Comment:
```suggestion
    def ngen_cal_model_post_iteration(self, iteration: int, info: JobMeta) -> None:
```

Reply:
History has gotten out of sync, this function was changed to `...iteration_finish` several commits back 😄